Rdmp.Core/DataExport/DataExtraction/Pipeline/Destinations/ExecuteFullExtractionToDatabaseMSSql.cs

```diff
@@ -471,8 +471,7 @@ public override GlobalReleasePotential GetGlobalReleasabilityEvaluator(IRDMPPlat
         protected override void TryExtractLookupTableImpl(BundledLookupTable lookup, DirectoryInfo lookupDir,
             IExtractionConfiguration requestConfiguration,IDataLoadEventListener listener, out int linesWritten, out string destinationDescription)
         {
-            var tbl = lookup.TableInfo.Discover(DataAccessContext.DataExport);
-            var dt = tbl.GetDataTable();
+            var dt = lookup.GetDataTable();
```

Comment:
OK to use using instead of .Dispose to simplify this slightly?

Reply:
Yup think thats fine yeah :)